Gemfile

```diff
@@ -6,6 +6,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}" }
 
 gemspec
 
+gem "fontist", github: "fontist/fontist", ref: "e79ed1e"
```

Comment:
Should we move it to gemspec?

Reply:
Absolutely, but maybe fontist gem hasn't been released yet?

Reply:
Yap, like Ron mentioned both of these gems are still in development, so for now testing it out with the edge version but we've to change those once things got finalized :)

Reply:
Done now :)